config/locales/client.en.yml

```diff
@@ -29,13 +29,14 @@ en:
           theme:
             label: "Theme"
             instructions: "Theme with page assets"
-            select: "Select theme"
+            select: "Select theme…"
           groups:
             label: "Groups"
             instructions: "Limit access to group(s)"
           category:
             label: "Category"
             instructions: "Connect page to category (requires child pages)"
+            select: "Select category…"
```

Comment:
@angusmcleod I added here a new key to display when no category is selected, but I am not familiar with the i18n process in this repository. Should I use Crowdin (as there is a `crowdin.yml` configuration file) or is there any other method to align the rest of the translations? Should I just remove the other translation files as they are essentially copies of the English (default) one?